lexibank_robinsonap.py

```diff
@@ -12,6 +12,8 @@
 class Language(BaseLanguage):
     # the tokens used as language identifiers by Robinson and Holton
     Token = attr.ib(default=None)
+    Latitude = attr.ib(default=None)
```

Comment:
I think, these attributes are no longer needed with lexibank 2.1. I have to remove them from the data I patron as well...

Reply:
Yes! sorry -- I didn't see your commit come in, so I've deleted this one!